src/bootstrap/builder.rs

```diff
@@ -1063,6 +1063,14 @@ impl<'a> Builder<'a> {
             }
         }
 
+        match mode {
+            Mode::Rustc | Mode::Codegen => {
+                rustflags.arg("-Zunstable-options");
+                rustflags.arg("-Wrustc::internal");
```

Comment:
I think this'll get passed to crates.io deps for the compiler which'll presumably break things?

Reply:
Heh I would agree with you if things were broken, but the bootstrap proceeded so I don't think it'll break too many things. I'm not really sure where these come from but I'm assuming it's a somewhat magical rustc thing.

Reply:
I think the CI failure is due to this change? (https://rust-lang.github.io/rust-log-analyzer/log-viewer/#https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/7750/logs/49)

Reply:
That's an in-tree we can tweak, and it's only in tests, so it doesn't mean the whole bootstrap is busted. Just needs a few `#![allow]` in the tests which I'll be pushing up soon.